db/migrate/20161118210435_add_question_number.rb

```diff
@@ -0,0 +1,6 @@
+class AddQuestionNumber < ActiveRecord::Migration
+  def change
+    add_column :questions, :number, :integer, unique: true
```

Comment:
Does this handle the case where `number` can appear multiple times as long as it's unique within a survey? That was my understanding for number.